task7/index.js

```diff
@@ -0,0 +1,45 @@
+const obj = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+const div = document.getElementById("js-parent");
+
+const loading = () => {
+    const loadingPlace = document.createElement("div");
+    const gif = document.createElement('img');
+    loadingPlace.id = "loadingPlace";
+    gif.src = "loading-circle.gif";
+    div.appendChild(loadingPlace).appendChild(gif);
+};
+    
+const getData = new Promise((resolve) => {
+    loading();
+    setTimeout(() => {
+        resolve(obj);
+    },2000);
+});
+
+getData.then(() => {
+    document.getElementById("loadingPlace");
+    loadingPlace.remove();
+});
```

Comment:
I'm getting a warning in codesandbox.
<img width="628" alt="スクリーンショット 2021-07-25 15 05 37" src="https://user-images.githubusercontent.com/43788628/126889467-8c8ca4e7-0da3-492b-ab40-51cbebe9b0b6.png">

IMO
```suggestion
getData.then(() => {
    const loadingPlace = document.getElementById("loadingPlace");
    loadingPlace.remove();
});
```

Reply:
Sorry for your inconvenience.
I thought that the variable once defined should be omitted, so it was written like this.
I'll be careful.